test/rest-api/response-patching.mocks.ts

```diff
@@ -28,6 +28,19 @@ const { start } = composeMocks(
       )
     },
   ),
+
+  rest.post(
+    'https://jsonplaceholder.typicode.com/posts',
```

Comment:
I understand this is a placeholder response from a third-party service. Do you know if there's any end of life period for such responses? I'm worried that this placeholder will auto-close after some timeout and tests would start to fail.

For the cases when interaction with third-party APIs is unavoidable (like response patching) I usually pick a public API that's unlikely to fail (i.e. GitHub API). I'd advise to do the same for this test.

Reply:
We will stick with jsonplaceholder for now and add an express server for testing response patching later for the purposes resiliency.

Reply:
We are going to leave this in place for now until we implement an express server for more robust healthchecks.